pages/[username].js

```diff
@@ -45,12 +45,30 @@ export async function getServerSideProps(context) {
     log.error(e, `cannot strip markdown for: ${username}`);
   }
 
+  // fetching data from api
+  let apiEvents = {};
+  try {
+    const res = await fetch(
+      `https://api.eddiehub.io/github/${profile.username}`
```

Comment:
I think we should move this to an environment variable as sometimes we might need to point it to local